src/layouts/EditContactUs.jsx

```diff
@@ -76,6 +78,34 @@ const enumSection = (type, args) => {
   }
 };
 
+const displayDeletedFrontMatter = (deletedFrontMatter) => {
+  let displayText = ''
+  const { contacts, locations } = deletedFrontMatter
+  contacts.forEach((contact, i) => {
+    if (!isEmpty(contact)) {
+      displayText += `<br/>In Contact <code>${i+1}</code>: <br/>`
+      contact.content.forEach((obj) => {
+        const [key, value] = Object.entries(obj)[0]
+        if (value) {
+          displayText += `    <code>${key}</code> field, <code>${JSON.stringify(value)}</code> is removed <br/>`
+        }
+      })
+    }
+  })
+  displayText += ''
+  locations.map((location, i) => {
+    if (!isEmpty(location)) {
+      displayText += `<br/>In Location <code>${i+1}</code>: <br/>`
+      Object.entries(location).forEach(([key, value]) => {
+        if (value?.length) {
+          displayText += `    <code>${key}</code> field, <code>${JSON.stringify(value)}</code> is removed </br>`
+        }
+      })
+    }
+  })
```

Comment:
Since the order of fields on the contact us page is `locations` followed by `contacts`, I think it might be more consistent to show the deleted items in that order as well

Reply:
That's a good idea, I've swapped it here 5ca1033